src/App.tsx

```diff
@@ -20,17 +27,38 @@ class App extends React.Component<{}, State> {
       './img/9.png',
       './img/10.png',
     ],
+    step: defaultStep,
+    frameSize: defaultFrameSize,
+    itemWidth: defaultItemWidth,
+    animationDuration: defaultAnimationDuration,
+  };
+
+  handlerChangeSettings = (e: React.ChangeEvent<HTMLInputElement>) => {
+    const { name, value } = e.target;
+
+    this.setState(prev => ({
+      ...prev,
+      [name]: value,
+    }));
   };
 
   render() {
-    const { images } = this.state;
+    const { images, step, frameSize, itemWidth, animationDuration } =
+      this.state;
 
     return (
       <div className="App">
         {/* eslint-disable-next-line */}
         <h1>Carousel with {images.length} images</h1>
```

Comment:
```suggestion
        <h1 data-cy="title">Carousel with {images.length} images</h1>
```
Consider adding all needed data attributes to pass tests